core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -187,7 +187,7 @@ private[spark] class Executor(
 
   val isolatedSessionCache = CacheBuilder.newBuilder()
     .maximumSize(100)
-    .expireAfterAccess(5, TimeUnit.MINUTES)
+    .expireAfterAccess(10, TimeUnit.MINUTES)
```

Comment:
Actually, I already faced the limitation here when the test takes longer than 5 mins. I increased it as a workaround for now. We should fix them before the release.